components/org.wso2.carbon.business.rules.web/src/utils/AuthManager.jsx

```diff
@@ -153,4 +194,51 @@ export default class AuthManager {
     static deleteSessionCookie(name) {
         document.cookie = name + '=;expires=Thu, 01 Jan 1970 00:00:00 UTC;path=' + appContext;
     }
+
+    /**
+     * Set a cookie with given name and value assigned to it.
+     * @param {String} name : Name of the cookie which need to be set
+     * @param {String} value : Value of the cookie, expect it to be URLEncoded
+     * @param {number} validityPeriod :  (Optional) Validity period of the cookie in seconds
+     * @param {String} path : Path which needs to set the given cookie
+     * @param {boolean} secured : secured parameter is set
+     */
+    static setCookie(name, value, validityPeriod, path = "/", secured = true) {
+        let expires = '';
+        const securedDirective = secured ? "; Secure" : "";
+        if (validityPeriod) {
+            const date = new Date();
+            date.setTime(date.getTime() + validityPeriod * 1000);
+            expires = "; expires=" + date.toUTCString();
+        }
+        document.cookie = name + "=" + value + expires + "; path=" + path + securedDirective;
+    }
+
+    /**
+     * Get JavaScript accessible cookies saved in browser, by giving the cooke name.
+     * @param {String} name : Name of the cookie which need to be retrived
+     * @returns {String|null} : If found a cookie with given name , return its value,Else null value is returned
+     */
+    static getCookie(name) {
+        let pairs = document.cookie.split(";");
+        let cookie = null;
+        for (let pair of pairs) {
+            pair = pair.split("=");
+            let cookie_name = pair[0].trim();
```

Comment:
shall we continue using camelcase?

Reply:
This function (`getCookie(name)`) doesn't contain `cookie_name` anymore. _(See https://github.com/wso2/carbon-analytics/blob/master/components/org.wso2.carbon.business.rules.web/src/utils/AuthManager.js#L303)_